Assets/Scripts/HUD/EndOfDayReport.cs

```diff
@@ -51,13 +56,53 @@ void Start()
             MostPlayedMinigame.text = $"{returnPrefabTaskNameInDutch(playerReportData.GetTheMostPlayedMiniGameName(out playNr)).ToString()} : {playNr} keer.";
             timeBonus.text += $"{remainingTime} seconden resterend";
         */
-
-
+        writePlayFabData();
+      
         Cursor.lockState = CursorLockMode.None;
         Cursor.visible = true;
+        Time.timeScale = 0f;
         
     }
 
+    private void writePlayFabData()
+    {
+        playFabManager.SendLeaderBoard(playerReportData.GetTheSuccessfulMinigameNumber());
+        playFabManager.WriteCustomPlayerEvent("Distance_travelled_per_game",new Dictionary<string, object> {
```

Comment:
Would it be possible if you make a method out of these? And then the different values as that method's params?

Reply:
That is in the end WriteCustomPlayerEvent. I created that method to make  events it is not built in